pkg/results/result.go

```diff
@@ -185,6 +198,31 @@ func (l Labels) MarshalJSON() ([]byte, error) {
 	return json.Marshal(aux)
 }
 
+// String prints all result labels
+func (l Labels) String() string {
```

Comment:
FYI: String methods didn't change, I just moved them around so the methods are all right next to their respective types...